test/ecto/query/builder/filter_test.exs

```diff
@@ -4,6 +4,14 @@ defmodule Ecto.Query.Builder.FilterTest do
   import Ecto.Query.Builder.Filter
   doctest Ecto.Query.Builder.Filter
 
+  def strip_heredoc(string) do
```

Comment:
There is no need for strip_heredoc. Just build the parts using `<>` and we will be fine!